Misc/NEWS.d/next/Core_and_Builtins/2024-11-04-21-58-20.gh-issue-125038.kGhWOS.rst

```diff
@@ -0,0 +1,3 @@
+Adds :opcode:`CHECK_ITERABLE` instruction. It checks that TOS is iterable or
```

Comment:
What is "TOS"? I would just say something generic like "value" here.

Reply:
Top of stack. Probably better to expand it though as that term doesn't appear in the public-facing documentation.